<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Чоловічі імена</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="list_male">
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QRadioButton" name="radio_male_a">
              <property name="text">
               <string>Множина А</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">button_group_male</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_male_b">
              <property name="text">
               <string>Множина В</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">button_group_male</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="save_male">
              <property name="text">
               <string>Зберегти</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Множина В</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="list_b"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>Зберегти</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>Очистити</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Жіночі імена</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="list_female">
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QRadioButton" name="radio_female_a">
              <property name="text">
               <string>Множина А</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">button_group_female</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radio_female_b">
              <property name="text">
               <string>Множина В</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">button_group_female</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="save_female">
              <property name="text">
               <string>Зберегти</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Множина А</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="list_a"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>Зберегти</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>Очистити</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="information">
   <property name="text">
    <string>Про студента</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Зберегти до файлу</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="button_group_male">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
  <buttongroup name="button_group_female">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
